python/ray/scripts/scripts.py

```diff
@@ -741,82 +738,107 @@ def start(
+                    )
+                    cli_logger.print(
+                        cf.bold("  ray stop"),
+                    )
+                    cli_logger.print(
+                        cf.bold("  {}=true ray start"),
+                        ray_constants.ENABLE_RAY_CLUSTERS_ENV_VAR,
+                    )
+                    cli_logger.newline()
+                    cli_logger.print(
+                        "`{}=true` must also be passed to any Ray clients.",
+                        ray_constants.ENABLE_RAY_CLUSTERS_ENV_VAR,
+                    )
+                cli_logger.newline()
+            else:
+                cli_logger.print("To add another node to this Ray cluster, run")
+                # NOTE(kfstorm): Java driver rely on this line to get the address
+                # of the cluster. Please be careful when updating this line.
                 cli_logger.print(
-                    "This Ray runtime only accepts connections from local host."
+                    cf.bold("  ray start --address='{}'"),
+                    bootstrap_address,
                 )
+                cli_logger.newline()
+            if ray_constants.ENABLE_RAY_CLUSTER:
                 cli_logger.print(
-                    "To accept connections from remote hosts, "
-                    "specify a public ip when starting"
+                    "To connect to this Ray cluster, run `ray.init()` as usual:"
                 )
+                with cli_logger.indented():
+                    cli_logger.print("{} ray", cf.magenta("import"))
+                    cli_logger.print(
+                        "ray{}init()",
+                        cf.magenta("."),
+                    )
+                cli_logger.newline()
                 cli_logger.print(
-                    "the head node: ray start --head --node-ip-address=<public-ip>."
+                    "To connect to this Ray instance from outside of "
+                    "the cluster, for example "
                 )
-            cli_logger.newline()
-            cli_logger.print("Alternatively, use the following Python code:")
-            with cli_logger.indented():
-                cli_logger.print("{} ray", cf.magenta("import"))
-                # Note: In the case of joining an existing cluster using
-                # `address="auto"`, the _node_ip_address parameter is
-                # unnecessary.
                 cli_logger.print(
-                    "ray{}init(address{}{}{})",
-                    cf.magenta("."),
-                    cf.magenta("="),
-                    cf.yellow("'auto'"),
-                    ", _node_ip_address{}{}".format(
-                        cf.magenta("="), cf.yellow("'" + node_ip_address + "'")
+                    "when connecting to a remote cluster from your laptop, "
+                    "make sure the"
+                )
+                cli_logger.print(
+                    "dashboard {}is accessible and use Ray jobs. For example:",
```

Comment:
```suggestion
                    "dashboard {}is accessible and use the Ray Jobs API. For example:",
```

Reply:
Don't feel strongly about this, but this is what we call it in the docs currently